packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -9,37 +9,45 @@ description: With this guide you'll learn how to get started with the iOS app
 
 Bring your CodeSandbox projects with you wherever you go with just your iPhone or iPad. First, download the CodeSandbox app from the [App Store](https://apps.apple.com/us/app/codesandbox/id1423330822) and log in using your CodeSandbox account. If you don't have one, you can quickly sign up using a GitHub, Google, or Apple account.
 
-For the full CodeSandbox experience, we recommend linking your GitHub account. This will allow you to import your GitHub repositories and contribute to them faster.
+> 💡 For the full CodeSandbox experience, we recommend linking your GitHub account. This will allow you to import your GitHub repositories and contribute to them faster.
 
 ### Select the Appropriate Workspace
 
-When you log in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar. On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button in the screen's top left corner.
+When you log in, your personal workspace will be selected as the default. If you're part of a team, you can easily switch to your team's workspace by selecting it from the sidebar.
+
+On an iPhone, you can access the screen to switch workspaces by tapping the **Back** button in the screen's top left corner.
+
+> 💡 [Create a team](https://codesandbox.io/dashboard?create_team=true) to fully experience how CodeSandbox can [accelerate your team's workflow](https://codesandbox.io/develop-full-stack-apps).
```

Comment:
As the user will be browsing these docs from the iOS app I would avoid having links to parts of the web client as it can make the user leave the iOS app entirely.

Reply:
I completely forgot about that! Will revert it. Thanks!